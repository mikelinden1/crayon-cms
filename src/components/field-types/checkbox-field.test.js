import CheckboxField from './checkbox-field';

const props = {
    name: 'test',
    label: 'Test',
    value: true,
    onChange: () => {},
    disabled: false
};

it('CheckboxField matches snapshot', () => {
    const wrapper = shallow(<CheckboxField {...props} />);
    expect(wrapper).toMatchSnapshot();
});
