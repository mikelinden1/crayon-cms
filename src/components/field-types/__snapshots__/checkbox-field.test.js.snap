// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CheckboxField matches snapshot 1`] = `
<div
  className="custom-control custom-checkbox"
  style={
    Object {
      "margin": "30px 0",
    }
  }
>
  <input
    checked={true}
    className="custom-control-input"
    disabled={false}
    id="test"
    name="test"
    onChange={[Function]}
    type="checkbox"
  />
  <label
    className="custom-control-label"
    htmlFor="test"
  >
    Test
  </label>
</div>
`;
